eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -248,6 +248,11 @@
           "new": "class com.azure.messaging.eventhubs.EventData",
           "superClass": "com.azure.core.models.MessageContent",
           "justification": "To support Schema Registry Avro Serializer's signature, EventData extends from MessageContent."
+        },
+        {
```

Comment:
Hi @JonathanGiles , I add a new method to the AmqpTrait, the revapi reports a problem, I found there is no regex for ignoring problem for AMQP related class, so I add a config for this modify.